.github/workflows/deploy-npm-package.yaml

```diff
@@ -49,3 +49,11 @@ jobs:
           npm install --include=dev
           npm publish --tag develop
         fi
+
+    - name: Hit endpoint and build from scratch pre-built image
+      run: |
+        curl --fail-with-body -X POST \
+        https://dev.polyapi.io/functions/server/prebuilt-base-image \
+        -H "Authorization: Bearer ${{ secrets.POLY_SUPER_ADMIN_USER_KEY }}" \
+        -H "Content-Type: application/json" \
+        -d '{}'
```

Comment:
we might need a delay before we run this step. But I'm hoping the natural latency let's this pick up the newly published version on npm!